src/fixtures/Observation.yml

```diff
@@ -16,7 +16,8 @@ items:
     status: '@status*'
     pullusAge: '@pullusAge*'
     accuracyOfPullusAge: '@accuracyOfPullusAge*'
-    geographicalCoordinates: '{{address.latitude}} {{address.longitude}}'
+    latitude: '{{address.latitude}}'
```

Comment:
What exactly returns faker? strings or numbers?

Reply:
Sorry, I don't know, but it isn't a problem. Now we store it as numeric type in DB

Reply:
Will answer myself. It creates number type